html/semantics/forms/constraints/form-validation-validity-patternMismatch.html

```diff
@@ -17,14 +17,15 @@
         {conditions: {pattern: null, value: "abc"}, expected: false, name: "[target] The pattern attribute is not set"},
         {conditions: {pattern: "[A-Z]+", value: ""}, expected: false, name: "[target] The value attibute is empty string"},
         {conditions: {pattern: "[A-Z]{1}", value: "A"}, expected: false, name: "[target] The value attribute matches the pattern attribute"},
-        {conditions: {pattern: "[A-Z]+", value: "\u0041\u0042\u0043"}, expected: false, name: "[target] The value(ABC) in unicode attribute matches the pattern attribute"},
+        {conditions: {pattern: "[A-Z]+", value: "\x41\x42\x43"}, expected: false, name: "[target] The value(ABC) in unicode attribute matches the pattern attribute"},
         {conditions: {pattern: "[a-z]{3,}", value: "ABCD"}, expected: true, name: "[target] The value attribute mismatches the pattern attribute"},
         {conditions: {pattern: "[A-Z]+", value: "ABC123"}, expected: true, name: "[target] The value attribute mismatches the pattern attribute even when a subset matches"},
         {conditions: {pattern: "(abc", value: "de"}, expected: false, name: "[target] Invalid regular expression gets ignored"},
+        {conditions: {pattern: "[(]", value: "x"}, expected: false, name: "[target] Invalid `v` regular expression gets ignored"},
```

Comment:
@annevk This one + line 42 now fail with `u` but pass with `v`. We could add more such tests if you like, but it’d be duplicating more of the `.breaking-change-from-u-to-v` tests in `html/semantics/forms/the-input-element/pattern_attribute.html`. WDYT?